packages/mcp-server/src/server/store/user-library-metadata-store.test.ts

```
@@ -21,21 +28,44 @@ const {
   USER_LIBRARY_METADATA_FILENAME_SUFFIX,
 } = await import('./user-library-metadata-store.js')
+const { createIsolatedDb } = await import('./db/test-helpers.js')
+
+interface StoreFixture {
+  db: Awaited<ReturnType<typeof createIsolatedDb>>['db']
+  tempDir: string
+}
+
+const test = baseTest.extend<{ store: StoreFixture }>({
+  store: [
+    async ({}, use) => {
```

Biome (2.4.13)

[error] Unexpected empty object pattern.

(lint/correctness/noEmptyPattern)